### PR TITLE
Fix syntax error in Jinja template

### LINKED ISSUE #24: Push MOTD location to clients
Push the MOTD url to the clients.  Then try to open it:

```
push "setenv-safe motd https://github.com/cisagov/cool-system/blob/develop/motd.md#welcome-to-cisas-cloud-oriented-operations-lab-cool"
```

```
route-up "/usr/bin/open ${OPENVPN_motd}"
```

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description
The syntax was wrong, which angers OpenVPN.  This should correct it. 

## 💭 Motivation and Context
It dun broke in staging. 

## 🧪 Testing

Tested by fixing syntax in staging.  Worked.  @dav3r connected.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 👍 👍

**Review by jsf9k — APPROVED** [review]
Glory be!

**Review by mcdonnnj — APPROVED** [review]
🔨 Fix it.

File: cloudinit/openvpn-config.tpl.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ write_files:
       # value is 0 seconds, which disables this feature.  If the optional bytes
       # parameter is included, exit if less than bytes of combined in/out
       # traffic are produced on the tun/tap device in n seconds. In any case,
-      # OpenVPN’s internal ping packets (which are just keepalives) and TLS
+      # OpenVPN's internal ping packets (which are just keepalives) and TLS
```
Comment (jsf9k) [inline cloudinit/openvpn-config.tpl.yml:69]: Those non-ASCII characters were a problem after all!

**Pull request merged**